public/index.html

```diff
@@ -24,6 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <script src="https://kit.fontawesome.com/29bccbf253.js" crossorigin="anonymous"></script>
```

Comment:
I love using Font Awesome too but I recently discovered React Icons (includes Font Awesome icons too)! 

Here is the documentation : https://react-icons.github.io/react-icons/icons?name=fc